app/models/vote.rb

```diff
@@ -0,0 +1,4 @@
+class Vote < ApplicationRecord
+    belongs_to :user
```

Comment:
I suggest that you also add a validation that the combination user_id and work_id are unique.